rasa_core/training/dsl.py

```diff
@@ -8,6 +8,7 @@
 import json
 import logging
 import os
+import rasa_nlu
```

Comment:
we need to specifically import this submodule, let's do:
`from rasa_nlu import utils as nlu_utils`